test/cli_run_dns_test.go

```diff
@@ -66,3 +66,72 @@ func (suite *PouchRunSuite) TestRunWithBridgeNetwork(c *check.C) {
 
 	c.Assert(res.Stdout(), check.Equals, hostRes.Stdout())
 }
+
+// TestRunWithDNSFlags tests DNS related flags.
+func (suite *PouchRunDNSSuite) TestRunWithDNSFlags(c *check.C) {
+	cname := "TestRunWithDNSFlags"
+
+	res := command.PouchRun("run", "--name", cname,
```

Comment:
since `res` not used, how about `command.PouchRun(xx).(c, icmd.Success)`

Reply:
Nope. `res` is used in line 84.

Reply:
OK, got it , my mistake